sauce_connect_setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Would prefer things like this live in a `scripts` folder, instead of directly in root.